com.microsoft.mrtk.core/Utilities/Handedness.cs

```diff
@@ -6,27 +6,30 @@
 namespace Microsoft.MixedReality.Toolkit
 {
     /// <summary>
-    /// Defines in which hand a controller is currently being used.
+    /// Represents handedness; as a bitflag, it can represent left-handedness,
+    /// right-handedness, both, or neither.
     /// </summary>
     [Flags]
     public enum Handedness
     {
         /// <summary>
-        /// No hand specified by the SDK for the controller.
+        /// No handedness. Represents a non-handed controller or object.
         /// </summary>
-        /// <remarks>
-        /// This value is used for controllers such as a gamepad or a mouse.
-        /// </remarks>
         None = 0 << 0,
 
         /// <summary>
-        /// The controller is identified as being provided in a Left hand
+        /// The user's left hand or controller.
         /// </summary>
         Left = 1 << 0,
 
         /// <summary>
-        /// The controller is identified as being provided in a Right hand
+        /// The user's right hand or controller.
         /// </summary>
-        Right = 1 << 1
+        Right = 1 << 1,
+
+        /// <summary>
+        /// Both, or either, hands.
+        /// </summary>
+        Both = Left | Right,
```

Comment:
I feel like there was some reason we only ported over Left/Right/None, but I can't remember exactly (and I can't remember why we'd do that but leave them as flags anyway...). I remember we explicitly didn't want to bring over _both_ `Both` and `Any` (or `Other`)
 https://github.com/microsoft/MixedRealityToolkit-Unity/blob/4d71cf198304b830ee566c652ae1373bfdde1efa/Assets/MRTK/Core/Definitions/Utilities/Handedness.cs#L28-L40

but i no longer remember the specifics...I'm good with the change but wanted to call this out in case it jogged any memories

Reply:
Handedness as it relates to all controllers, not just hands should support left, right and other as Trackables and other non-handed "trackables" do not have a hand to speak of (unless they are licked up ;D )

Reply:
I recall that I personally had confusion over the implementation of Both and Any. To me, Both means that the controller _requires_ / is intended to be held in both hands whereas any means it does not matter which hand is used / provided.

Reply:
Both was simply intended (effectively) to be Either hand,  so the action is supported if used in the Left hand or right hand (both).
As opposed to Left, meaning the action is limited to only the left hand.

Reply:
That said, defining Both (defined as Left | Right vs a separate value) negates the ability to say "i dont care which hand"

Reply:
Agreed, the naming really should have been "Either", however, at the time we had to go with the common standards coming out of Khronos :S